metals/src/main/scala/scala/meta/internal/metals/ReferenceProvider.scala

```diff
@@ -88,12 +88,28 @@ final class ReferenceProvider(
     }
   }
 
+  def referencesForWildcardImport(
+      sym: String,
+      source: AbsolutePath,
+      directlyImportedSymbols: Set[String],
+  ): List[String] = {
+    semanticdbs.textDocument(source).documentIncludingStale match {
+      case Some(doc) =>
+        doc.occurrences
+          .map(_.symbol)
```

Comment:
```suggestion
          .map(_.symbol).distinct
```
maybe to avoid doing `onwerChain` multiple times?